src/components/ThreeDNodeTree/ThreeDNodeTree.tsx

```diff
@@ -80,7 +82,13 @@ class ThreeDNodeTree extends React.Component<NodeTreeProps, NodeTreeState> {
   }
 
   async componentDidMount() {
-    const threeDNodes = await this.context!.viewer3D.listRevealNodes3D(
+    this.client = this.context('ThreeDNodeTree')!;
```

Comment:
we talked that you should use `this.context(Component.displayName || '')!` instead
any reason to hardcode this string?

Reply:
Actually no- I'll fix this.

Reply:
you have this issue in other places as well

Reply:
Fixed.

However, in `Model3DViewer` when used like `this.client = this.context(ThreeDNodeTree.displayName || '')!;` with `static displayName - 'Model3DViewer';` set inside class like done in `SVGViewer`, `yarn lint` gives the following error for the `componentDidMount` method:

```
Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed.
```

Reply:
so is it `ThreeDNodeTree` or `Model3DViewer` ?

Reply:
It was `Model3DViewer`